docs/pages/release_notes.md

```diff
@@ -21,6 +21,12 @@ This is a {{ site.pmd.release_type }} release.
 
 ### API Changes
 
+* In PMD 7, The `language` attribute will be required on all `rule` elements that
```

Comment:
I'll add the keyword "ruleset" - possibly as a section heading. Otherwise it is not clear, what we are talking about here...
Otherwise this is a good explanation, thanks!